docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -9,50 +9,50 @@
     "slug": "aperturedb"
   },
   "oss": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "community",
+    "source_url": "https://hub.docker.com/r/aperturedata/aperturedb-community",
+    "comment": "ApertureDB community is a great way to test all the database features, locally."
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "C++"
```

Comment:
c++ (we standardize on lowercase, see the validation run failure here https://github.com/superlinked/VectorHub/actions/runs/12007888772/job/33563981447?pr=532 )